test/utils/shippable/other.sh

```diff
@@ -5,6 +5,12 @@ set -o pipefail
 retry.py apt-get update -qq
 retry.py apt-get install -qq \
     shellcheck \
+    g++-4.9 \
```

Comment:
looks like merge botch

I've amended the commit and re-rebased on top of current devel and updated.